README.md

```diff
@@ -252,6 +252,7 @@ Databases and data stores with C APIs.
 * [kdtree][337] - Simple library for working with KD-trees. [``BSD-3-Clause``][BSD-3-Clause]
 * [libavl][156] - Library containing a range of self-balancing binary
   trees. [``GPL-3.0-or-later``][GPL-3.0-or-later]
+* [libds](https://github.com/recp/ds) - Common Data Structures and Algorithms. [``MIT``][MIT]
```

Comment:
You still have an inline link. Observe ``libavl`` above for an example of how to proceed.

Reply:
Fixed. But may I ask what is the advantage of number [542] instead of link? If there is a good reason, I may use that syntax in my READMEs :)